spec/items_repo_spec.rb

```diff
@@ -121,12 +121,12 @@
     it 'can update an item' do
       original_time = ir.find_by_id(263567475).updated_at
       attributes = {
-        unit_price: BigDecimal(379.99, 5),
```

Comment:
Did we decide to stop using bigdecimal here because its already in our unit price attribute or for another reason?

Reply:
yes, it was already being called in the method.